Makefile

```diff
@@ -151,6 +151,16 @@ docker-build: test
 docker-push:
 	docker push ${IMG}
 
+PLATFORMS ?= linux/amd64,linux/arm64
+.PHONY: docker-buildx
+docker-buildx: ## Build docker image for the manager with cross-platform support
+	docker buildx build \
+		--file Dockerfile \
+		--platform=$(PLATFORMS) \
+		--tag ${IMG} \
+		--push \
+		${CURDIR}
+
```

Comment:
Can you do this using docker build only? Should just need to add the --platform arg to the make hook on 143, similar to what you have here